### PR TITLE
Renamed 'nodejs' to contrib

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
@jhadvig PTAL

**Comment by soltysh** [comment]
LGTM.

**Comment by mfojtik** [comment]
this is pending new base image to be available on hub

File: 0.10/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,10 @@
 FROM openshift/base-centos7
 
+# This image provides a Node.JS environment you can use to run your Node.JS
+# applications.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
```
Comment (jhadvig) [inline 0.10/Dockerfile]: :+1:

File: 0.10/Dockerfile.rhel7 [hunk 2]
```diff
@@ -1,5 +1,10 @@
 FROM openshift/base-rhel7
 
+# This image provides a Node.JS environment you can use to run your Node.JS
+# applications.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
```
Comment (jhadvig) [inline 0.10/Dockerfile.rhel7]: :+1:  :+1:

**Comment by jhadvig** [comment]
LGTM

**Comment by mfojtik** [comment]
[test]

**Comment by soltysh** [comment]
@mfojtik why there's no test results? The same for https://github.com/openshift/sti-ruby/pull/4 ?

**Comment by mfojtik** [comment]
@soltysh they are part of the "Pull request successfully merged and closed" box, for some reason GH wipes that out... check the Jenkins, there should be job with the same ID as this PR

**Comment by soltysh** [comment]
@mfojtik how come we have this working in other projects' PRs

**Pull request merged**